pkgs/tools/misc/fluent-bit/default.nix

```diff
@@ -13,11 +13,17 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake flex bison ];
 
+  buildInputs = [ systemd ];
```

Comment:
This package seems to have some basic darwin support (and `platforms.unix`), so I assume that `systemd` should only be a build-input for `stdenv.isLinux`?